src/openapi.json

```diff
@@ -0,0 +1,566 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "Philanthropy Data Commons API",
+    "description": "An API for a common data platform to make the process of submitting data requests to funders less burdensome for organizations seeking grants.",
+    "version": "0.0.3"
+  },
+  "components": {
+    "parameters": {
+      "canonical_field_label": {
+        "name": "canonical_field_label",
+        "description": "The unique name of a canonical PDC field.",
```

Comment:
I'm wondering if the ERD overloaded this field.

Label elsewhere is meant to be the human readable "label" (I tend to agree it should be unique but I also wonder if it must) -- we might also want a canonical field *code* which is intended to be more meaningful than an int ID but also used in semantic contexts (e.g. when defining a mapping between an application schema field and a canonical field).

Thoughts?

(genuine side note that you can ignore: this kind of question might expose a flaw in our plan of "define the whole api spec, and all migrations, at once and then implement the endpoint one by one" since it asks us to think about the implementation for ALL endpoints at once and potentially answer a whole bunch of these kinds of nitty gritty questions outside the context of that implementation.  Whoops!)

Reply:
I like your side note, basically we can defer or delay this question.